utils/clean_txt.py

```diff
@@ -2,59 +2,108 @@
 """A command line utility to clean text files generated from PDFs for Praxis-NLP."""
 import argparse
 from pathlib import Path
-from sys import stderr
 import re
 
-BRACKET_REGEX = re.compile(r"\s?\[\w*\]\s?|\s?\(\w*\)\s?")
-BIB_REGEX = re.compile(r"(Bibliography)|(Works Cited)|(Source Extracts)", re.IGNORECASE)
+# match expressions in square or round brackets containing alphanumeric
+# characters or spaces
+BRACKET_REGEX = re.compile(r'\s?\[(\w|\s|,)*\]|\s?\((\w|\s|,)*\)')
```

Comment:
I think you're missing some grouping here, around the regex which matches the parenthesized expression. W

Reply:
Because you're trying to match the left expression or the right expression, not a close bracket or one space.

Reply:
I'm gonna go with citation needed on that one - given operator precedence, this should be fine https://stackoverflow.com/a/36870447/11035406

Reply:
Is the | supposed to bind to the expression that matches the entire parenthesized expression? Because I think rn it's binding just to the \] and the \s? in accordance to the thing you just sent.

Reply:
Just figured it out. Resolved.